apps/src/Packager.cpp

```diff
@@ -46,40 +44,52 @@ struct VolumeInfo {
 
 static bool DoAnalyze{true};
 
-auto GetVolumeInfo(const fs::path& slicePath) -> VolumeInfo;
+auto GetVolumeInfo(const po::variables_map& parsed) -> VolumeInfo;
 void AddVolume(vc::VolumePkg::Pointer& volpkg, const VolumeInfo& info);
 
 auto main(int argc, char* argv[]) -> int
 {
     ///// Parse the command line options /////
-    // All command line options
     // clang-format off
     po::options_description options("Options");
     options.add_options()
         ("help,h", "Show this message")
         ("volpkg,v", po::value<std::string>()->required(),
-           "Path for the output volume package")
-        ("material-thickness,m", po::value<double>(),
-           "Estimated thickness of a material layer (in microns). Required "
-           "when making a new volume package.")
-        ("slices,s", po::value<PathStringList>(),
-            "Path to input slice data. Ends with prefix of slice images or log "
-            "file path. Can be specified multiple times to add multiple "
-            "volumes.");
+           "Path for the output volume package.");
 
-    // Useful transforms for origin adjustment
-    po::options_description extras("Metadata");
-    extras.add_options()
+    po::options_description volpkg_metadata("Volpkg metadata");
+    volpkg_metadata.add_options()
         ("name", po::value<std::string>(),
             "Set a descriptive name for the VolumePkg. Default: Filename "
-            "specified by --volpkg");
-    // clang-format on
+            "specified by --volpkg")
+        ("material-thickness,m", po::value<double>(),
+           "Estimated thickness of a material layer (in microns). Required "
+           "when making a new volume package.");
+
+    po::options_description volume_options("Volume");
+    volume_options.add_options()
+        ("slices,s", po::value<std::string>(),
+            "Path to input slice data. Ends with prefix of slice images or log "
+            "file path. Required when making a new volume.")
+        ("volume-name,n", po::value<std::string>(),
+            "Descriptive name for the volume. Required when making a new "
+            "volume.")
+        ("voxel-size-um,u", po::value<double>(),
+            "Voxel size of the volume in microns (e.g. 13.546). Required when "
+            "making a new volume.")
+        ("flip,f", po::value<std::string>()->default_value("none"),
+            "Flip options: Vertical flip (vf), horizontal flip (hf), both, "
+            "z-flip (zf), all, [none].")
+        ("compress,c", po::value<std::string>()->default_value("n"),
+            "Compress slice images? [yN].");
```

Comment:
Let's just make this a normal flag:
```suggestion
        ("compress,c", "Compress slice images");
```